### ISSUE TITLE
Update to Yarn v2

### ISSUE THREAD

**Issue by nihaals** [thunderstore-io/thunderstore-ui#1]
Blocked by:
- [x] https://github.com/dependabot/dependabot-core/issues/1297
  - dependabot might not be *required* but it's a nice to have

Decisions:
- [ ] Whether to use [zero-installs](https://yarnpkg.com/features/zero-installs)

---

We can also migrate to [Plug 'n' Play](https://yarnpkg.com/features/pnp)
